android/app/build.gradle

```diff
@@ -141,17 +141,18 @@ android {
 
 repositories {
     maven {
-        url "https://raw.github.com/socialvibe/truex-tv-integrations/android"
+        url "https://s3.amazonaws.com/android.truex.com/maven"
     }
 }
 
 dependencies {
+    implementation project(':react-native-video')
```

Comment:
can't come from a maven repo somewhere?

Reply:
I think the dependency is managed by npm. Here we are just linking it. 
Indeed these codes are autogenerated with the `react-native link ` command that is recommended in the documentation.
https://github.com/react-native-community/react-native-video#android-installation